content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/utxo-advanced-ledger-extensions/ledger-extensions-api-reference.md

```diff
@@ -394,13 +394,13 @@ class ExampleOwnableContract : DelegatedContract<ExampleOwnableContract.ExampleO
 
     object Update : ExampleOwnableContractCommand {
```

Comment:
Still not right - if you chose indentation of 4 spaces then every nesting level should have 4 spaces more relative to the previous scope/block of code. So it should look like this:
```
    object Update : ExampleOwnableContractCommand {
  
        override fun getContractStateType(): Class<ExampleOwnableState> {
            return ExampleOwnableState::class.java
        }

        override fun verify(transaction: UtxoLedgerTransaction) {
            OwnableConstraints.verifyUpdate(transaction, contractStateType)
        }
    }
```